recipes/poco/all/patches/1.8.1.patch

```diff
@@ -0,0 +1,59 @@
+--- cmake/PocoMacros.cmake
++++ cmake/PocoMacros.cmake
+@@ -265,7 +265,7 @@
+ 
+ if (MSVC)
+ # install the targets pdb
+-  POCO_INSTALL_PDB(${target_name})
++  # POCO_INSTALL_PDB(${target_name})
+ endif()
+   
+ endmacro()
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -204,7 +204,7 @@
+ 
+ find_package(OpenSSL)
+ if(OPENSSL_FOUND)
+-    include_directories("${OPENSSL_INCLUDE_DIR}")
++    include_directories("${OpenSSL_INCLUDE_DIR}")
```

Comment:
Probably not needed if we are using targets

Reply:
Using current conan, it is required.
The cmake_find_package generator created the `OpenSSL_INCLUDE_DIR` variable, not the `OPENSSL_INCLUDE_DIR` variable.
`include_directories` fails with the following error if given an empty argument:
```
CMake Error at source_subfolder/CMakeLists.txt:207 (include_directories):
  include_directories given empty-string as include directory
```

Reply:
My point is that we can remove this line, the information from the include-dirs is contained in the target `OpenSSL::OpenSSL`